src/base/forms/_base.scss

```diff
@@ -31,6 +32,22 @@ input[type="checkbox"] {
 }
 // scss-lint:enable QualifyingElement
 
+.form-group {
+	&.has-error {
+		.checkbox {
+			color: $text-color;
+		}
+	}
+
+	.checkbox {
+		&.checkbox-bold {
+			label {
+				font-weight: 700;
+			}
+		}
```

Comment:
That is a new feature.

We will need to validate the use case and test locally.

And that new feature need to be described. It could be a simple paragraph explaining when to use and when not to use.

Reply:
@duboisp What would your thoughts be about dropping the ``checkbox-bold`` class and making something along the lines of ``.wb-frmvld .form-group:not(ul) .checkbox label`` bold by default?

IMO it'd be more elegant to code and in line with the ``form-group`` class' intended use case (its purpose is to [space-out label/fields pairs](https://getbootstrap.com/docs/3.4/css/#forms-example)):
* Tying its selector to the form validation plugin's class (``.wb-frmvld``) would avoid conflicts with [WET](https://wet-boew.github.io/v4.0-ci/demos/tables/tables-fltrs-en.html)/[GCWeb](https://wet-boew.github.io/GCWeb/components/gc-chckbxrdio/gc-chckbxrdio-en.html)'s table filter examples.
* The ``:not(ul)`` selector would prevent it from clashing with the form validation plugin's [stacked checkbox examples](http://localhost/wet-boew/unmin/demos/formvalid/formvalid-en.html#fruits1a).

The downside is that I don't know if any pre-existing forms are already doing anything that might get picked up by my selector. Might need to something along the lines of https://github.com/wet-boew/wet-boew/pull/9147#issuecomment-956577561 to find out.

Reply:
I've updated the documentation to try addressing the first two todos. Tried to keep it short and sweet.